addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -390,6 +390,13 @@ function factory(dependencies) {
         initActiveId: attr({
             default: 'mail.box_inbox',
         }),
+        /**
+         *  Determine whether the thread from init active id was already
+         *  opened or not.
+         */
+        initThreadOpened: attr({
```

Comment:
Could you rename the field to start with "is" to convey that it is a boolean?
You may also want to use "open" (adjective) instead of "opened" (the past form of the verb).